### ISSUE TITLE
Information missing about organizations' participation in certificate checking for booking

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2218]
## Where to find the issue

- https://www.coronawarn.app/en/blog/2021-12-20-cwa-2-15/ "Corona-Warn-App simplifies certificate checks when booking tickets"
- https://www.coronawarn.app/de/blog/2021-12-20-cwa-2-15/ "Corona-Warn-App vereinfacht die Zertifikatsprüfungen bei der Ticketbuchung"

## Describe the issue

The announcement for CWA 2.15 includes extensive information about "certificate checks when booking tickets".

There is no reference given for organizations who would like to take part in this service.

## Suggested change

1. Publish information for organizations who want to take part in the certificate checking process, so that they know what they should do.

2. For end-users give some indication if this service is already live or just in preparation. Which organizations are already providing this service? Edit: This is done through the update to https://www.coronawarn.app/en/blog/2021-12-20-cwa-2-15/ on January 20, 2022, which says: "In Germany, no provider of an online verification service has been approved yet."

---
Internal Tracking ID: [EXPOSUREAPP-11110](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11110)

**Comment by dsarkar** [comment]
@MikeMcC399 Currently the project partners are in contact with providers in order to integrate them in the infrastructure of the ticketing/validation service.

**Comment by Ein-Tim** [comment]
FYI: https://www.t-systems.com/de/en/newsroom/news/corona-validation-service-475486

**Comment by MikeMcC399** [comment]
The question has also been raised now in https://github.com/corona-warn-app/cwa-documentation/issues/781.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for raising this issue.
Internal Tracking ID: [EXPOSUREAPP-11110](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11110)

---
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
https://www.coronawarn.app/en/faq/#val_service_basics says:

"**Is there an approval process for DCC validation service providers?**

Interested companies can go through a test procedure at the BMG. The procedure is open to all providers."

This is part of the same issue that there are no documented concrete points of contact (e.g. website / e-mail) to start the process of becoming an accredited validation service provider.

**Comment by GisoSchroederSAP** [comment]
I'm going to clarify the contact details for participation and integration of the validation service. The results will then be added to the existing communication in FAQ and/or blog.

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

Please consider creating a new page like https://www.coronawarn.app/en/rat-partner/ for "Become a rapid test partner!" either located on https://www.coronawarn.app or somewhere else, for organisations wishing to offer a validation service (like T-Systems today) or organisations which want to take part in the certificate checking process.

**Comment by GisoSchroederSAP** [comment]
That's exactly my intention, as we already mentioned this request in our daily sync meeting swith T-Systems. So, they should be prepared, especially as I can name at least three interested integration partners already.

**Comment by GisoSchroederSAP** [comment]
First response: We now have at least an official email recipient, that is available to gather the enquiries for the validation service: 
DCC-validation@t-systems.com

I still asked for a webpage with some more explanation about the process and available interfaces. Let's see...

Happy 2022 to you all!

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 
There is also some information on the webpage https://landing.ticket.io/dein-ticketing/covid-validation about integration with T-Systems and CWA.

**Comment by Ein-Tim** [comment]
@GisoSchroederSAP 

> Happy 2022 to you all!

Thanks, the same back to you! "Guten Rutsch!" 😉

**Comment by MikeMcC399** [comment]
The second suggestion in this issue

"For end-users give some indication if this service is already live or just in preparation. Which organizations are already providing this service?"

has been implemented. https://www.coronawarn.app/en/blog/2021-12-20-cwa-2-15/ has been amended:

"**Important update from January 10, 2022**: The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, no provider of an online verification service has been approved yet. Tests are currently being carried out, the results of which are being awaited for the time being. Online verification services will not be available in practice until these tests have been evaluated and the Federal Commissioner for Data Protection and Freedom of Information (BfDI) has provided advice. With version 2.15, however, the CWA is ready to support corresponding verification services as soon as providers of such a service are approved."

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

You wrote:
> First response: We now have at least an official email recipient, that is available to gather the enquiries for the validation service:
DCC-validation@t-systems.com

- If this is official, then why is the PR https://github.com/corona-warn-app/cwa-website/pull/2266 to publish the e-mail address on hold?

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

It seems there has been no update on the domain of the validation service since Jan 2022. Has this functionality now been put on ice? It would be good to get a new status.

**Comment by GisoSchroederSAP** [comment]
In fact, last week I asked who is offering this service and it turns out: All the ice is melted. Strictly speaking, there is no provider that still offers such a validation service anymore.

This status given, I don't expect any update on the topic anymore, at least not before the pandemic situation will be re-evaluated maybe in late summer or fall this year.

Does this answer your quesion(s) around the validation feature?

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

Thank you for the update. If there will be no new blog post on this subject in the next weeks and months, then perhaps the text in the FAQs should be updated accordingly?

**Comment by Ein-Tim** [comment]
@GisoSchroederSAP Maybe you could provide a new update which I could add to #2734

**Comment by Ein-Tim** [comment]
PR https://github.com/corona-warn-app/cwa-website/pull/2734 has been merged. Can this issue be closed?

**Comment by MikeMcC399** [comment]
The currently published information is as follows:

---
https://www.coronawarn.app/en/blog/2021-12-20-cwa-2-15/ "Corona-Warn-App simplifies certificate checks when booking tickets" 

"Important update from January 10, 2022: The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, no provider of an online verification service has been approved yet. Tests are currently being carried out, the results of which are being awaited for the time being. Online verification services will not be available in practice until these tests have been evaluated and the Federal Commissioner for Data Protection and Freedom of Information (BfDI) has provided advice. With version 2.15, however, the CWA is ready to support corresponding verification services as soon as providers of such a service are approved."

---
https://www.coronawarn.app/en/faq/results/#val_service_basics

"UPDATE April 12, 2022
The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, no provider of an online verification service has been approved. With version 2.15, however, the CWA is ready to support corresponding verification services."

---

If an organization is interested in becoming part of an online verification service, there is no information that the project is on hold. This would need to be added explicitly to the published information, for instance as a statement in https://www.coronawarn.app/en/faq/results/#val_service_basics,  before this issue is resolved.

**Comment by Ein-Tim** [comment]
@GisoSchroederSAP What do you think? Adapt the update to include note that this project is on hold? Should this also be added to the FAQ?

**Comment by Ein-Tim** [comment]
@MikeMcC399 I think this issue is obsolete now and can be closed, or?

**Comment by MikeMcC399** [comment]
@Ein-Tim

> I think this issue is obsolete now and can be closed, or?

You say that you think the issue is obsolete, although you did not give your reason. Perhaps you think that the statement from @GisoSchroederSAP in https://github.com/corona-warn-app/cwa-website/issues/2218#issuecomment-1094809124 "I don't expect any update on the topic anymore, at least not before the pandemic situation will be re-evaluated maybe in late summer or fall this year." makes it obsolete?

This issue was always about the public information available on https://www.coronawarn.app/ not just about the information that has been written in GitHub.

As I wrote in https://github.com/corona-warn-app/cwa-website/issues/2218#issuecomment-1097008183 "If an organization is interested in becoming part of an online verification service, there is no information that the project is on hold. This would need to be added explicitly to the published information, for instance as a statement in https://www.coronawarn.app/en/faq/results/#val_service_basics, before this issue is resolved."

I am however going to close this issue unresolved now as it seems there will not be any changed communication coming from the Open Source Team in the near future.

Instead I defer to the open issue https://github.com/corona-warn-app/cwa-documentation/issues/781 "How to implement the new check-in function for booking tickets" where @derweili asked how to participate. This is a better place to follow the state of the verification service project as far as IT implementers are concerned.

If the verification service project is revived due to a new need to check status as part of a ticket-booking process, then organizational participants would need advanced notice and preparation time. Probably nobody is in a position at the moment to predict if this is likely to happen later this year. This situation should be handled separately in any case.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> You say that you think the issue is obsolete, although you did not give your reason.

As you may have noticed, I just crawled trough all issues in this repo, asking for an update, same goes for this one.

Regarding https://github.com/corona-warn-app/cwa-website/issues/2218#issuecomment-1097008183 the real question to be asked is:
If there is an organization interested in providing a validation service, would this organization get approved? Or would all requests to provide a validation service be rejected.
It's probably the best to discuss this question in https://github.com/corona-warn-app/cwa-documentation/issues/781.

**Comment by Ein-Tim** [comment]
@MikeMcC399 I opened https://github.com/corona-warn-app/cwa-documentation/issues/885. Please let me know your thoughts, maybe you also disagree with me here. Anyways, enjoy Sunday!

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> I opened https://github.com/corona-warn-app/cwa-documentation/issues/885. Please let me know your thoughts, maybe you also disagree with me here.

- I closed this issue, which you said you thought was obsolete, because I did not expect any further answer in the next months from the Open Source Team on this subject. This was based on the statement from @GisoSchroederSAP in https://github.com/corona-warn-app/cwa-website/issues/2218#issuecomment-1094809124 "I don't expect any update on the topic anymore, at least not before the pandemic situation will be re-evaluated maybe in late summer or fall this year."

- This issue, by the way, was intended to be about information for online booking or check-in process service providers as well as for verification service providers.

- I don't expect there will be any new information which can be provided by the Open Source Team to your new issue https://github.com/corona-warn-app/cwa-documentation/issues/885, but good luck to you: I may be judging the situation incorrectly.
